datumaro/plugins/coco_format/extractor.py

```diff
@@ -124,6 +147,42 @@ def _load_items(self, loader):
 
         return items
 
+    def _load_panoptic_items(self, config, panoptic_images):
+        import numpy as np
```

Comment:
This import can be moved to the top of the file

Reply:
Fixed